misc/bazel/cargo-gazelle/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
```

Comment:
Non-blocking comment: I wouldn't mind having this in src/cargo-gazelle rather than in misc/bazel. I can see the argument both ways though.

Reply:
I slightly lean towards keeping it in `misc/bazel`, more than happy to move it in the future though!